### PR TITLE
Add a Process page to eips.ethereum.org

### PR DESCRIPTION
**Pull request by timbeiko**
The Ethereum Cat Herders have been trying to improve the legibility of the EIP process within Ethereum. This PR adds a Process page to eips.ethereum.org to describe the non-technical elements of the process. The contents of the page are inspired by this [Ethereum Magicians thread](https://ethereum-magicians.org/t/definitions-of-governance-layers/3054), specifically @bmann's [blog post](https://blog.bmannconsulting.com/ethereum-governance/) and [this tweetstorm](https://twitter.com/cyber_hokie/status/1112305608047427584).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,10 @@ <h1 class="page-heading">EIPs
   <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
   <a href="/last-call.xml"><img src="https://img.shields.io/badge/rss-Last Calls-red.svg" alt="RSS"></a>
 </h1>
-<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.</p>
+<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards. For more information on the EIPs process, see <a href="{{"process"|relative_url}}">Process</a> ({{site.pages|where:"type","Process"|size}}).</p>
```
Comment (Arachnid) [inline index.html]: What's the expression at the end for?

Comment (timbeiko) [reply index.html]: ... I'm honestly not sure. I tried to copy the format for other links, but haven't found any info on how to build this locally to try it out. See this for a similar example: https://github.com/ethereum/EIPs/pull/1932/files/38b0bc1d5f6420c629d006ae8c0c506395052789#diff-eacf331f0ffc35d4b482f1d15a887d3bR32

Comment (Arachnid) [reply index.html]: `({{site.pages|where:"type","Process"|size}})` outputs the number of pages with 'type: Process'. I don't think you need it here.

Building locally is just a matter of following the instructions for building a Github Pages Jekyll site.

Comment (timbeiko) [reply index.html]: Thanks! Managed to build locally. Will update soon.

**Comment by timbeiko** [comment]
Added some small changes to the `Process` page. I am not convinced whether this content is best as a page on eips.ethereum.org or if it should potentially be part of an existing EIP (i.e. EIP-1, EIP-233).

**Comment by nicksavers** [comment]
@timbeiko Could you look into the build failure?

**Comment by timbeiko** [comment]
@nicksavers done!

**Comment by axic** [comment]
I think it would make sense dropping most of the content from `README` in this PR, otherwise there will three source for explaining the process:
- EIP-1
- README
- Process paragraph on eips.ethereum.org

**Comment by timbeiko** [comment]
@axic I had a previous PR merged in to make the `README` and the index page on eips.ethereum.org more similar. Do you think it would be reasonable to perhaps move the contents here into the `README` and index page instead of bringing that content over in a new page? How about having it directly in EIP-1?

**Comment by axic** [comment]
I think any of those options are good as long as we do not have 3 sources of information :wink:

My preference would probably be an overhaul of EIP-1:
1) A chapter explaining the rules and processes.
2) A chapter providing a guide how to apply these rules in practice.

Point 2) is what the README and this PR tries to accomplish.

**Comment by timbeiko** [comment]
Closing in favor of https://github.com/ethereum/EIPs/pull/1991

**Pull request closed without merging**